src/seedu/addressbook/commands/TotalCommand.java

```diff
@@ -0,0 +1,30 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.List;
+
+
+/**
+ * Lists all persons in the address book to the user.
```

Comment:
Incorrect header comment. Remember to update the comment when you change the code (this looks like a copy-paste-forgot error)